src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -96,6 +98,7 @@ public ConfigurationManager(string metadataAddress, IConfigurationRetriever<T> c
         /// Instantiaties a new <see cref="ConfigurationManager{T}"/> that manages automatic and controls refreshing on configuration data.
         /// </summary>
         /// <param name="metadataAddress">the address to obtain configuration.</param>
+        /// <param name="configRetriever">the <see cref="IConfigurationRetriever{T}"/></param>
```

Comment:
Please capitalize "The". Please go through the whole file and capitalize all first words.